lib/kino_db/connection_cell.ex

```diff
@@ -465,8 +466,15 @@ defmodule KinoDB.ConnectionCell do
   end
 
   defp missing_dep(%{"type" => "athena"}) do
-    unless Code.ensure_loaded?(ReqAthena) do
-      ~s|{:req_athena, "~> 0.1"}|
+    cond do
+      not Code.ensure_loaded?(ReqAthena) ->
+        ~s|{:req_athena, "~> 0.1"}|
+
+      not Code.ensure_loaded?(Explorer) ->
+        ~s|{:explorer, "~> 0.9"}|
+
+      true ->
+        nil
     end
```

Comment:
If both are missing, we should suggest both, otherwise the user needs to reinstall twice.

```suggestion
    deps = [
      {ReqAthena, ~s|{:req_athena, "~> 0.1"}|},
      {Explorer, ~s|{:explorer, "~> 0.9"}|}
    ]
    
    deps = for {module, dep} <- deps, not Code.ensure_loaded?(module), do: dep
    
    if deps != [] do
      Enum.join(deps, ", ")
    end
```

Reply:
Makes sense. Thanks!